packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -1218,6 +1227,52 @@ function updateTransition(
   return [startTransition, isPending];
 }
 
+let clientId: number = 0;
+
+function mountUniqueId(): string | IdObject {
+  const [id, setId] = mountState(() => {
+    if (getIsHydrating()) {
+      return {
+        $$typeof: REACT_OPAQUE_OBJECT_TYPE,
+        _setId: () => {
+          setId(() => 'c_' + (clientId++).toString());
+        },
+        toString() {
+          throw Error(
+            'The object passed back from useUniqueID is meant to be passed through to ' +
+              'attributes only. Do not directly modify the output.',
```

Comment:
What does “don’t modify the output” mean? I’m just reading the value. Not modifying it.

Reply:
This should be an `invariant` since our error minification pipeline doesn't support error constructors, yet.